src/contexts/RainbowProvider/configs.tsx

```diff
@@ -0,0 +1,142 @@
+  name: string
+  wallet: Wallet
+  visible: boolean
+}
+
+const createWalletConfig = (name: string, walletFunction: () => Wallet, condition: boolean): WalletConfig => {
+  return {
+    name,
+    wallet: walletFunction(),
+    visible: condition,
+  }
+}
+
+export const scrollChain: Chain = {
+  id: CHAIN_ID.L2,
+  name: L2_NAME,
+  network: L2_NAME,
+  iconUrl: "https://scroll.io/logo.png",
+  iconBackground: "#fff",
+  nativeCurrency: {
+    decimals: 18,
+    name: L2_NAME,
+    symbol: ETH_SYMBOL,
+  },
+  rpcUrls: {
+    default: {
+      http: [RPC_URL.L2],
+    },
+    public: {
+      http: [RPC_URL.L2],
+    },
+  },
+  blockExplorers: {
+    default: { name: "Scrollscan", url: requireEnv("REACT_APP_EXTERNAL_EXPLORER_URI_L2") },
+  },
+}
+
+const sepoliaChain = produce(sepolia, draft => {
+  draft.rpcUrls.public.http = [RPC_URL.L1 as any]
+  draft.fees = {
+    // adopt MetaMask params
+    baseFeeMultiplier: 1,
+    defaultPriorityFee() {
+      return parseUnits("1.5", "gwei")
+    },
+  }
+})
+
+const mainnetChain = produce(mainnet, draft => {
+  draft.rpcUrls.public.http = [RPC_URL.L1 as any]
+  draft.fees = {
+    // adopt MetaMask params
+    baseFeeMultiplier: 1,
+    // defaultPriorityFee: parseUnits("0.05", "gwei"),
+    defaultPriorityFee() {
+      return parseUnits("0.05", "gwei")
+    },
+  }
+})
+
+const projectId = requireEnv("REACT_APP_CONNECT_WALLET_PROJECT_ID")
+
+const { chains, publicClient } = configureChains(
+  // ankr
+  [mainnetChain, sepoliaChain, scrollChain],
+  [
+    publicProvider(),
+    jsonRpcProvider({
+      rpc: chain => ({ http: chain.rpcUrls.default.http[0] }),
+    }),
+  ],
+)
+
+const walletConfigs: WalletConfig[] = [
+  createWalletConfig("Brave", () => braveWallet({ chains }), window.ethereum?.isBraveWallet === true),
+  createWalletConfig("MetaMask", () => metaMaskWallet({ chains, projectId }), window.ethereum?.isMetaMask === true),
+  createWalletConfig("Rainbow", () => rainbowWallet({ chains, projectId }), window.ethereum?.isRainbow === true),
+  createWalletConfig("Safe", () => safeWallet({ chains }), window.ethereum?.isSafeWallet === true),
+  createWalletConfig("Frame", () => frameWallet({ chains }), window.ethereum?.isFrame === true),
+  createWalletConfig("imToken", () => imTokenWallet({ chains, projectId }), window.ethereum?.isImToken === true),
+  createWalletConfig("Okx Wallet", () => okxWallet({ chains, projectId }), window.okxwallet?.isOKExWallet || window.okxwallet?.isOkxWallet === true),
+  createWalletConfig("Rabby", () => rabbyWallet({ chains }), window.ethereum?.isRabby && !window.ethereum?.isMetaMask === true),
+  // Add any additional wallets here
+]
+
+const activeWallets: Wallet[] = walletConfigs.filter(wallet => wallet.visible).map(wallet => wallet.wallet)
+
+const Wallets = [
+  // TODO: rainbowkit/injectedWallet.ts "Browser Wallet" and "injectedWallet.svg" -> need to detect automaticlly
+  injectedWallet({ chains }),
+  coinbaseWallet({ appName: "Scroll", chains }),
```

Comment:
can we place the active wallet at the forefront? and do we still need to permanently display the Coinbase wallet?